app/views/components/_govuk_input.html.erb

```diff
@@ -66,7 +66,7 @@
         { 'govuk-input--error' => local_assigns[:errorMessage] }
       ),
       id: local_assigns[:id],
-      name: local_assigns[:name],
+      name: local_assigns[:key],
```

Comment:
This component does need to take `name` AFAICS? https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/input/input.yaml#L6-L9